tests/optim/test_jacobian.py

```diff
@@ -117,6 +119,26 @@ def forward(self, inputs):
         J = pp.optim.functional.modjac(invnet, input=inputs, **jackwargs)
         assert not pp.hasnan(J)
 
+    def test_lietensor_jacfwd(self):
+        pose = pp.randn_SE3(1).to(device)
+        points = torch.randn(1, 3).to(device)
+        def func(pose, points):
+            return pose @ points
+        out = func(pose, points)
+
+        try: # the torch behavior since ver 2.0.0
+            jac_func = jacfwd(func)
+            jac = jac_func(pose, points)
+            raise AssertionError('should not reach here')
+        except RuntimeError as e:
+            assert 'shapes cannot be multiplied' in str(e)
+
+        with WrappableLT():
+            jac_func = jacrev(func)
```

Comment:
Is there a reason that on Line 130 we used `jacfwd`, but on Line 137 here we used `jacrev`?

Reply:
jacrev is the only thing we can support for now. jacfwd needs us to implement jvp in SE3_Act. jvp is also briefly discussed in https://pytorch.org/docs/stable/notes/extending.html

Reply:
Thanks for the link!